packages/office-ui-fabric-react/src/components/Button/Button.tsx

```diff
@@ -19,7 +25,7 @@ export interface IButtonState {
 
 export class Button extends BaseComponent<IButtonProps, IButtonState> implements IButton {
```

Comment:
Are you referring to the class? Because yes, I do have a ButtonBase that is extended by ButtonDefault.